app/helpers.php

```diff
@@ -177,3 +177,61 @@ function cachet_redirect($name, $parameters = [], $status = 302, $headers = [],
         return app('redirect')->to($url, $status, $headers);
     }
 }
+
+if (!function_exists('datetime_to_moment')) {
+    /**
+     * Convert PHP datetimes to moment.js formats.
+     *
+     * Thanks to http://stackoverflow.com/a/30192680/394013
+     *
+     * @param string $format
+     *
+     * @return string
+     */
+    function datetime_to_moment($format)
+    {
+        $replacements = [
```

Comment:
I didn't end up using this, but I like it, so I'm keeping it because I probably will use this at some point.